DataManagementSystem/private/FTS3Utilities.py

```diff
@@ -326,9 +334,14 @@ def _randomServerPolicy( self, _attempt ):
   @staticmethod
   def _getFTSServerStatus( ftsServer ):
     """ Fetch the status of the FTS server from RSS """
-    pub = RPCClient( 'ResourceStatus/Publisher' )
-    res = pub.getElementStatuses( 'Resource', ftsServer, None, None, None, None )
 
+    res = getFTS3ServerName(ftsServer)
+    if not res['OK']:
+      return res
+    ftsServerName = res['Value']
```

Comment:
I can't figure out from here what's `ftsServer` and what's `ftsServerName`, can you explain?

Reply:
our configuration looks like this:
  FTSEndpoints
  {
    FTS3
    {
      LCG.CERN.ch = https://fts3-pilot.cern.ch:8446
    }
  }
ftsServer is https://fts3-pilot.cern.ch:8446 and ftsServerName is LCG.CERN.ch. RSS can only be queried using the elementName (LCG.CERN.ch), previously the ftsServer url was passed instead of the elementName which returned an error that no FTS server was found in RSS, even-tough an FTS server with the name LCG.CERN.ch was registered.

Reply:
then maybe rather than looking up as you do in `getFTS3ServerName`, one should do differently and store a dict `{name:url}` rather than a list `[url]` ? I do not remember all the code so I am not sure how much work it would be to refactor it this way.

Reply:
@chaen a lot of files need to be re-factored and ftsSeverName is not needed in most parts of the code, so i think we should leave it the way it is now...

Reply:
So much for a clean code... :-)
This system is not used in prod anywhere now. If we want to do deep changes, now is the best time. 
Also, not sure what the impact can be, but we may have cases (we do in LHCb) where we use several times the same URL with different names